### ISSUE TITLE
16bit dependencies

### ISSUE THREAD

**Issue by tixdev** [otya128/winevdm#1321]
I'm running a 16-bit application with otvm, however, there's an issue when the application try to invoke odbc.dll library that is compiled in 16-bit. 

As otvdm converts instructions from 16 to 32 bits, it attempts to access the library in an 'illegal' manner, causing error. 

There's a possibility to exclude certain calls to this DLL from the conversion process?

**Comment by cracyc** [comment]
There are other programs which have no problem using odbc.  More detail about the problem and preferably a trace are needed.

**Comment by tixdev** [comment]
Some odbc calls after added some file to the windows or dll dir now works, but other operations with odbc not, here a complete session until will got error, maybe i'm using an invalid files structure.

<img width="330" alt="image" src="https://github.com/otya128/winevdm/assets/1364365/0baa41d4-dadf-4c5f-a57d-d796c46100c2">

<img width="631" alt="image" src="https://github.com/otya128/winevdm/assets/1364365/6942f410-278b-4943-910e-d9e667d2f18a">

[Logfile.CSV](https://github.com/otya128/winevdm/files/12786601/Logfile.CSV)

**Comment by cracyc** [comment]
Really need a wine trace.  See https://github.com/otya128/winevdm/issues/226#issuecomment-779702542

**Comment by tixdev** [comment]
Ok sorry :)

[trace.txt](https://github.com/otya128/winevdm/files/12788241/trace.txt)

**Comment by cracyc** [comment]
The trace shows a different error, "0084:Call USER.430: LSTRCMP(295f123e "dbWeb Administrator Error Handler",139f1a56 "Error IM002: ODBC data source not found") ret=165f:00f5 ds=1207".   Do you have the connection string set in the odbc admin control panel?

**Comment by tixdev** [comment]
Yes, I created the entries with ODBC Administrator (32-bit) in the System DSN section. Subsequently, I copied the odbc.ini file to the otvm/windows directory; otherwise, the application wouldn't recognize them. By doing this, the application can now list the sources. However, I encounter an exception when trying to add data to a source. Looking at the file, my suspicion is that it refers to a 32-bit driver, or, it could be searching for a non-existent file or that some other files are missing, but i can't find in trace.

<img width="1010" alt="image" src="https://github.com/otya128/winevdm/assets/1364365/0f73bc74-e8e8-4c17-bc75-71424d7d094b">

<img width="472" alt="image" src="https://github.com/otya128/winevdm/assets/1364365/60f4cc88-c4cd-4f6b-b420-d8277b8290b7">

**Comment by cracyc** [comment]
A trace that has the error in the image is whats needed.  Since the last trace had the wrong error it doesn't help debugging.

**Comment by tixdev** [comment]
I recreated the trace file with the scenario mentioned above.

[trace.txt](https://github.com/otya128/winevdm/files/12799023/trace.txt)

**Comment by cracyc** [comment]
Try pointing the dsn at the version of odbcjt32.dll that comes with dbweb rather than with windows.  The version that comes with windows 11 is missing some functions that the 16bit odbc library uses.
